scripts/start/readCheckScenarioConfig.R

```diff
@@ -36,7 +36,7 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
   }
   nameisNA <- grepl("^NA$", rownames(scenConf))
   if (any(nameisNA)) {
-    warning("Don't use 'NA' as scenario name, you fool. Stopping now.")
+    warning("Do not use 'NA' as scenario name, you fool. Stopping now.")
   }
```

Comment:
Forgoing the contraction "don't", while still calling the user a fool, seems like an odd stylistic break. ;)